CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+### Version 0.12.1.beta
```

Comment:
Just a sidebar I usually don't do version bumps in feature PRs since something else could come along and get merged while a feature PR is in flight. I do :heart: keeping the changlog up to date, often will just make this top section something like `master` until we tag it.